babel/src/main/java/org/apache/calcite/sql/babel/SqlCharacterSetToCharacterSet.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql.babel;
+
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class SqlCharacterSetToCharacterSet extends SqlIdentifier {
```

Comment:
nit: missing class javadoc

Reply:
Also, are you sure that SqlIdentifier is the right class to inherit? I've seen SqlIdentifier be used mainly for things like column names, and this seems a little more complex than an identifier, more like a function call in disguise.

Reply:
Added javadoc in https://github.com/googleinterns/calcite/pull/84/commits/8e44505458d4554ffb3933a781c8dd8d57393deb

Reply:
SqlIdentifier would be appropriate, this is not a function call. Meanwhile, SqlTranslateUsingCharacterSet is a function call so it would extend SqlCall.